packages/imperative/src/config/__tests__/ProfileInfo.TeamConfig.unit.test.ts

```diff
@@ -1342,4 +1349,316 @@ describe("TeamConfig ProfileInfo tests", () => {
+                                    latestFrom: "Zowe Client App"
+                                }
+                            }
+                        },
+                        res: {
+                            success: true
+                        }
+                    }
+                );
+            });
+
+            it("does not update a profile type in the schema if the version is older", async () => {
+                expectAddToSchemaTester(
+                    { previousVersion: "2.0.0", schema: { title: "Mock Schema" } as any, version: "1.0.0" },
+                    {
+                        extendersJson: {
+                            profileTypes: {
+                                "some-type": {
+                                    from: ["Zowe Client App"],
+                                    version: "2.0.0",
+                                    latestFrom: "Zowe Client App"
+                                }
+                            }
+                        },
+                        res: {
+                            success: false
+                        }
+                    }
+                );
+            });
+
+            it("updates a profile type in the schema - version provided, no previous schema version", async () => {
+                expectAddToSchemaTester(
+                    { previousVersion: "none", schema: { title: "Mock Schema" } as any, version: "1.0.0" },
+                    {
+                        extendersJson: {
+                            profileTypes: {
+                                "some-type": {
+                                    from: ["Zowe Client App"],
+                                    version: "1.0.0",
+                                    latestFrom: "Zowe Client App"
+                                }
+                            }
+                        },
+                        res: {
+                            success: true
+                        }
+                    }
+                );
+            });
+
+            it("does not update the schema if schema version is invalid", async () => {
+                expectAddToSchemaTester(
+                    { previousVersion: "none", schema: { title: "Mock Schema" } as any, version: "1.0.0" },
+                    {
+                        extendersJson: {
+                            profileTypes: {
+                                "some-type": {
+                                    from: ["Zowe Client App"],
+                                    version: "1.0.0",
+                                    latestFrom: "Zowe Client App"
+                                }
+                            }
+                        },
+                        res: {
+                            success: true
+                        }
+                    }
+                );
+            });
+        });
+        describe("buildSchema", () => {
+            // TODO
```

Comment:
Is there an outstanding action associated with this todo?

Reply:
Yes, thanks for catching this - resolved in 10c2f9e 👍